test/plan_builder_spec.rb

```diff
@@ -275,6 +275,38 @@
             Stripe::Plans::METERED.put!
           end
 
+          it 'creates a tiered plan' do
+            Stripe::Plan.expects(:create).with(
+              :id => :tiered,
+              :currency => 'usd',
+              :product => {
+                :name => 'Tiered',
+                :statement_descriptor => nil,
+              },
+              :interval => 'month',
+              :interval_count => 1,
+              :trial_period_days => 0,
+              :usage_type => 'metered',
+              :aggregate_usage => 'max',
+              :billing_scheme => 'tiered',
+              :tiers => [
+                {
+                  :unit_amount => 1500,
+                  :up_to => 10
+                },
+                {
+                  :unit_amount => 1000,
+                  :up_to => 'inf'
+                }
+              ],
+              tiers_mode: 'graduated'
+            )
+            plan = Stripe::Plans::TIERED
+            puts plan
+            puts plan.valid?
+            puts plan.errors.full_messages
+            Stripe::Plans::TIERED.put!
+          end
 
```

Comment:
Please add a spec with an invalid tier. I'd love to make sure that validation errors on `BillingTier` end up raising `Stripe::InvalidConfigurationError`